torch/csrc/jit/runtime/symbolic_script.cpp

```diff
@@ -1440,6 +1440,12 @@ const std::vector<std::string> functions = {
                 return None, None
             return torch.ne(self, other), backward
 
+        def hardshrink(self, lambd: number):
+          def backward(grad_output):
+            mask = ((self > float(lambd)) | (self < -float(lambd))).type_as(self)
```

Comment:
why do you need to add the float call here ? this should work for number : https://github.com/pytorch/pytorch/blob/master/aten/src/ATen/native/native_functions.yaml#L6028

Reply:
`hardshrink` only works with float tensors. I figured it would be clearer this way.

Reply:
it only works with float tensors but you can pass in an integer for `lambd`. If you do we're reducing precision by casting it to float.

Reply:
Wouldn't we need to convert to a float anyway since the tensors can only be float?
Anyways, I removed it already.